### PR TITLE
update zywall and zyxelnwa detection #5343

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

Got test data for a Zyxel NWA unit so updated to have this detected correctly.

Fixes: #5432

### CODE REVIEW

File: includes/definitions/zywall.yaml [hunk 1]
```diff
@@ -6,7 +6,10 @@ over:
     - { graph: device_bits, text: Traffic }
 icon: zyxel
 discovery:
-    - sysObjectId:
-        - .1.3.6.1.4.1.890.1.6
-        - .1.3.6.1.4.1.890.1.15
+    -
+      sysObjectId:
+          - .1.3.6.1.4.1.890.1.6
+          - .1.3.6.1.4.1.890.1.15
```
Comment (murrant) [inline includes/definitions/zywall.yaml]: My thought was this:
```
   -
      sysObjectId:
          - .1.3.6.1.4.1.890.1.6
          - .1.3.6.1.4.1.890.1.15
      sysDescr_regex:
          - '/^ZyWALL/
```

I'm not a fan of negative matches, they make a big mess.

File: includes/definitions/zyxelnwa.yaml [hunk 2]
```diff
@@ -5,4 +5,4 @@ type: network
 icon: zyxel
 discovery:
     - sysDescr_regex:
-        - '/^NWA-/'
+        - '/^NWA/'
```
Comment (murrant) [inline includes/definitions/zyxelnwa.yaml]: Can we combine this with a sysObjectId?

**Comment by laf** [comment]
I've updated them but I honestly don't think we have enough test data to do it this way and could just lead to more broken discovery.

**Comment by murrant** [comment]
@laf Possibly, any way you could put together that callback for sysDescr and sysObjectId?  I won't have any hacking time till maybe tonight and I'll have to familiarize myself with the callback stuff.

**Comment by laf** [comment]
@murrant callback?

**Comment by murrant** [comment]
@laf the data collection thingy in `includes/callback.php` :)

**Comment by laf** [comment]
Is this waiting for us to get more test data?

**Comment by murrant** [comment]
Yeah, but we can probably merge.  I'll go through them when we have data.

**Pull request merged**